### PR TITLE
DELETE endpoint added for removing comment via id - TASK 12

### PR DESCRIPTION
**Pull request by philjacks**
- endpoint added for removing comment via id
- status code 201 tested
- errors 400 & 404 tested

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -447,3 +447,18 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("Status 204 - should return a no content success status when deleting a comment by id", () => {
+    return request(app).delete("/api/comments/18").expect(204);
+  });
```
Comment (cjplatten) [inline __tests__/endpoints.test.js:454]: Can you think of a way to test if the comment has been deleted successfully?

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -447,3 +447,18 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+
+  test("Status 400 - should return a bad request error if an invalid data type is passed as a param", () => {
+    return request(app)
+      .delete("/api/comments/beans")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request");
+      });
+  });
```
Comment (cjplatten) [inline __tests__/endpoints.test.js:463]: What happens if the id is valid but does not exist?

**Review by cjplatten — COMMENTED** [review]
Please see the comments, good work so far!

File: __tests__/endpoints.test.js (hunk 2 above)
Comment (philjacks) [reply __tests__/endpoints.test.js:472]: extra test added

**Pull request merged**